### PR TITLE
[BN-1028]: PostgreSQL driver

### PR DESCRIPTION
**Pull request by szymonmiks-netguru**
### Task:
https://netguru.atlassian.net/browse/BN-1028

### Desc:
Added driver and performance test for postgresql

### CODE REVIEW

**Review by socek — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -37,3 +38,12 @@ services:
       image: postgres:10
       ports:
         - 5432:5432
+
+  postgres_test_database:
+    <<: *env_file
+    container_name: postgres_test_database
+    volumes:
+      - postgres_test_database_volume:/var/lib/postgresql/data/
+    image: postgres:10
```
Comment (daniel-tomaszuk) [inline docker-compose.yml:47]: Some newer one could be used. postgres:11 is available; version 12 might not be available yet.

File: fixtures/postgresql_schema.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
+CREATE OR REPLACE FUNCTION trigger_set_timestamp()
+RETURNS TRIGGER AS $$
```
Comment (daniel-tomaszuk) [inline fixtures/postgresql_schema.sql:2]: Nice.

File: src/drivers/postgres/impls/cqrs/queries.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+from src.drivers.postgres.impls.cqrs.common import execute_sql
+
+
+def get_all_customers():
+    sql = 'select * from customers'
+    return execute_sql(sql)
+
+
+def get_customer_by_id(customer_id):
+    sql = f"select * from customers where id=%s"
```
Comment (daniel-tomaszuk) [inline src/drivers/postgres/impls/cqrs/queries.py:10]: I'm wondering about sql injections a little. I guess we don't have to consider it in RnD. :dunno:

**Review by daniel-tomaszuk — APPROVED** [review]
lgtm

**Pull request merged**